src/System.Private.ServiceModel/tools/test/SelfHostWcfService/TestResources/EndpointResource.cs

```diff
@@ -110,5 +111,17 @@ private Uri BuildUri(ResourceRequestContext context)
             builder.Scheme = Protocol;
             return builder.Uri;
         }
+
+        private static string Getfqdn()
+        {
```

Comment:
I don't believe it is correct to assume the Bridge is on an Azure VM in this case. I think this value should be configurable, though we can certainly default to cloudapp.net in TestProperties. We allow configuration of host name, ports, etc. and this feels like the same. If there is a way to determine this value dynamically it would be best, but configurable is a better option if we can't. TestProperties was invented to allow smart defaults with the ability to override via MsBuild or environment variables. I think we should make this a new TestProperties value. I can make a quick PR to make this easier if it helps the multi-machine testing effort. In fact, we could do that PR *after* you merge and don't need to block on the discussion.